core/src/saros/net/xmpp/contact/IContactsUpdate.java

```diff
@@ -0,0 +1,28 @@
+package saros.net.xmpp.contact;
+
+import java.util.Optional;
+import saros.net.xmpp.XMPPConnectionService;
+
+/** Interface for receiving Contact Updates from {@link XMPPConnectionService}. */
+public interface IContactsUpdate {
+
+  /** List of possible update types. */
+  enum UpdateType {
+    ADDED,
+    CONNECTED,
```

Comment:
ONLINE / OFFLINE ?

Reply:
Added Javadoc for clarification, now I would say to not change it because online / offline sounds more like a update on a specific contact.